superset/config.py

```diff
@@ -496,6 +504,8 @@ class D3Format(TypedDict, total=False):
     "PLAYWRIGHT_REPORTS_AND_THUMBNAILS": False,
     # Set to True to enable experimental chart plugins
     "CHART_PLUGINS_EXPERIMENTAL": False,
+    # Set to True to limit the number of users who can certify datasets
+    "LIMIT_CERTIFICATION": False,
```

Comment:
Couldn't this be controlled via Superset's/FAB's existing permission/view model? This feels more consistent/coherent than creating more custom logic in the security manager.

Reply:
Thanks. I'll give that another look. I couldn't see within that section how to target individual fields. We want all users to be able to edit their own datasets to add things like metrics etc. but limit who can add this field.